src/services/helpers/service.js

```diff
@@ -1,7 +1,11 @@
-const nodemailer = require('nodemailer');
+const request = require('request-promise-native');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'request-promise-native'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4776801)